### PR TITLE
feat : added cart-management page

### PR DESCRIPTION
**Pull request by sach10-create**

### CODE REVIEW

**Review by sanjitsarkar — APPROVED** [review]
Looks good

**Review by Omkar-Ghate — APPROVED** [review]
Looks good for me.

File: css/cart-management.css [hunk 1]
```diff
@@ -0,0 +1,196 @@
+.ecom-cart-container {
+  display: grid;
+  grid-template-areas:
+    "header"
+    "heading"
+    "products"
+    "products"
+    "products"
+    "products"
+    "footer";
```
Comment (iRohitGaur) [inline css/cart-management.css:10]: no need to repeat "products" 4 times. You can give 1fr to it instead

File: store/pages/cart-management.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  <link rel="stylesheet" href="../../css/cart-management.css">
+</head>
+
+<body>
+  <div class="ecom-cart-container">
+    <header class="ecom-header .header-shadow">
+      <div class="header-container-1">
+        <div class="left-container">
+          <div class="logo-container">
+            <img src="../../assets/nova-logo-dark.png" alt="Logo" class="logo" />
```
Comment (iRohitGaur) [inline store/pages/cart-management.html:23]: you can do "/assets/..." instead

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge with updates

**Pull request merged**